helm/charts/amplication/values-common.yaml

```diff
@@ -45,12 +52,24 @@ cp-helm-charts:
     enabled: false
 
 postgres:
-  install: false
+  install: true
+  postgresqlPassword: postgres
+  postgresqlPostgresPassword: postgres
+  postgresqlDatabase: amplication
+  fullnameOverride: postgresql
+  primary:
+    service:
+      type: LoadBalancer
+      annotations:
+        service.beta.kubernetes.io/aws-load-balancer-backend-protocol: tcp
+        service.beta.kubernetes.io/aws-load-balancer-ssl-cert: arn:aws:acm:us-east-1:407256539111:certificate/28d98c3b-f831-4cfd-b71b-577f9f7a5d45
+        service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "443"
+        external-dns.alpha.kubernetes.io/hostname: postgres.amplication-dev.com
 
 amplication-client:
   image:
     repository: 407256539111.dkr.ecr.us-east-1.amazonaws.com/amplication-client
-    tag: "release-0-13-0" # Need to change for dev.
+    tag: "release-0-14-1" # Need to change for dev.
```

Comment:
Can you explain what this value means? Is it still needed?

Reply:
It's just for my testing when I'm deploying the helm-chart manually, it doesn't affect anything else it can be deleted.

Reply:
Okay, let's make sure we delete it in this PR then please.

Reply:
Deleted